server/src/test/java/keywhiz/service/daos/SecretSeriesDAOTest.java

```diff
@@ -263,4 +263,54 @@ private void assertListDoesNotContainSecretsWithIds(List<SecretSeries> secrets,
   private int tableSize() {
     return jooqContext.fetchCount(SECRETS);
   }
+
+  @Test public void getMultipleSecretSeriesByNameReturnsOne() {
```

Comment:
nit: could we add a test that requesting two different secrets returns both secrets? (or update this test)